x/twap/listeners_test.go

```diff
@@ -42,7 +106,7 @@ func (s *TestSuite) TestSwapTriggeringTrackPoolId() {
 // TODO: Abstract this to be more table driven, and test more pool / block setups.
 func (s *TestSuite) TestSwapAndEndBlockTriggeringSave() {
```

Comment:
Do we benefit from testing both UniV2 and multi-asset pools here as well? (asking to learn, unsure if it would provide value)

Reply:
yes! These are WIP, I wanted to have them in separate PR so that the PR doesn't get too big